swiftgen.yml

```
@@ -0,0 +1,13 @@
+strings:
+ inputs: 
+  - CriticalMass/en.lproj/Localizable.strings
+ outputs:
+   - templateName: structured-swift4
+     output: CriticalMass/Generated/Strings.swift
+
+xcassets:
+  - inputs: 
+    - CriticalMass/Images.xcassets
+    outputs:
+      - templateName: swift4
+        output: CriticalMass/Generated/Images.swift
```

```suggestion
        output: CriticalMass/Generated/Images.swift
        
```

always end your file with a newline 🙃